src/components/feed-item/index.js

```diff
@@ -20,6 +22,7 @@ export const FeedItem = (props) => {
             creator: props.creator_id,
             objkt: String(props.id),
             displayView: true,
+            blur: nsfwList.includes(props.id.toString())
```

Comment:
Not tested yet but just looking at the edits can't the blur classname be passed one level up, on the container?

Reply:
Not exactly sure what you mean by that.

Reply:
Instead of passing down blur 2/3 levels, just adding the classname to the container in this file

Reply:
Oh. Ok. Originally, I thought that each of the different kinds of media might have a different kind of "blur" and even different levels of blur; that is why I passed it down to each component. I ended up doing the same for each kind of media, but still not sure if that is the best to go with. I could imagine that some formats might not even do a CSS blur but black out the content.

Reply:
Ok. I thought that "displayView' was already an example of a visual feature being passed down, so I followed that pattern. But I will change to apply the blur to the top level element since they end up being a CSS filter for all of them (at least for now).